index.html

```diff
@@ -4,21 +4,33 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
```

Comment:
Please add all the files from [this](https://github.com/microverseinc/linters-config/tree/master/html-css) linter config.

Reply:
Originally I thought I should put them in .gitignore, I have since removed them from the list so they will appear on github.